petasos/core/src/main/java/net/fhirfactory/pegacorn/petasos/core/tasks/management/participant/watchdogs/LocalPetasosTaskCleanupWatchdog.java

```diff
@@ -162,26 +198,102 @@ protected void taskContinuityWatchdog(){
                 }
             }
             if(unregisterTask){
-                getLogger().debug(".taskContinuityWatchdog(): Task {} is finalised, removing from shared cache... start", currentTaskId);
+                getLogger().debug(".actionableTaskCleanup(): Task {} is finalised, removing from shared cache... start", currentTaskId);
                 PetasosActionableTask unregisteredActionableTask = actionableTaskDM.removeTask(currentTaskId);
-                getLogger().debug(".taskContinuityWatchdog(): Task {} is finalised, removing from shared cache... done...");
-            }
-            if(getLogger().isDebugEnabled()){
-                getLogger().debug(".taskContinuityWatchdog(): Shared ActionableTaskCache size->{}", actionableTaskDM.getAllTaskIds().size());
+                getLogger().debug(".actionableTaskCleanup(): Task {} is finalised, removing from shared cache... done...");
```

Comment:
I think this section of logging was meant to go above in the if (unregisterTask)